### PR TITLE
[Merged by Bors] - chore(MeasureTheory/Integral/SetIntegral): fix lemma names

### PR DESCRIPTION
**Pull request by YaelDillies**
Rename:
* `integral_empty` -> `setIntegral_empty`
* `integral_union` -> `setIntegral_union`
* `integral_univ` -> `setIntegral_univ`
*  `setIntegral_congr` -> `setIntegral_congr_fun`
*  `setIntegral_congr₀` -> `setIntegral_congr_fun₀`
* `setIntegral_congr_set_ae` -> `setIntegral_congr_set`

From PFR

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean [hunk 1]
```diff
@@ -21,7 +21,7 @@ and is zero otherwise.
 
 Since `∫ x in s, f x ∂μ` is a notation, one can rewrite or apply any theorem about `∫ x, f x ∂μ`
 directly. In this file we prove some theorems about dependence of `∫ x in s, f x ∂μ` on `s`, e.g.
-`integral_union`, `integral_empty`, `integral_univ`.
+`integral_union`, `integral_empty`, `setIntegral_univ`.
```
Comment (grunweg) [inline Mathlib/MeasureTheory/Integral/SetIntegral.lean:24]: Can you rename `integral_union` and `integral_empty` as well? A follow-up PR is fine.

**Review by grunweg — COMMENTED** [review]
The new names make more sense than the old ones; thanks! This PR just contains the renames; tedious to review, but straight-forward. I have a follow-up request that can, well, be a follow-up PR. As CI is essentially green, let me already say
maintainer merge

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/MeasureTheory/Integral/SetIntegral.lean]: Well spotted! I hate those names too 😁

**Comment by grunweg** [comment]
Thanks, the new changes also look good to me!

**Pull request closed without merging**